### ISSUE TITLE
AW crashed unexpectedly after BSOD

### ISSUE THREAD

**Issue by KeksXtreme** [ActivityWatch/activitywatch#297]
## Describe the bug
When starting aw-qt.exe first a Windows notification pops up saying the program is launching, then aw-qt has a popup "Module aw-watcher-window quit unexpectedly". If I click the restart button on this popup the same thing happens and show details is empty. Cancel works, but then ActivityWatch (AW) is not running. There are three popups in total and the second one is titled aw-qt and says "Module aw-watcher-afk quit unexpectedly". There is no information in the show details box.
The third popup is titled aw-qt as well and says "Module aw-server quit unexpectedly" and the details are: 
`2019-08-09 18:19:57 [INFO ]: Using storage method: peewee  (aw_server.main:26)
2019-08-09 18:19:57 [INFO ]: Starting up...  (aw_server.main:31)
2019-08-09 18:19:57 [INFO ]: Using database file: C:\Users\<USERNAME>\AppData\Local\activitywatch\activitywatch\aw-server\peewee-sqlite.v2.db  (aw_datastore.storages.peewee:90)
2019-08-09 18:19:57 [INFO ]:  * Running on http://localhost:5600/ (Press CTRL+C to quit)  (werkzeug:88)
2019-08-09 18:20:01 [INFO ]: Received heartbeat after pulse window, inserting as new event. (bucket: aw-watcher-window_<MY_MACHINE>)  (aw_server.api:212)
`

## To Reproduce
(0. Crash PC??)
1. Open aw-qt.exe
2. Wait for the Windows popup saying AW is being launched
3. Wait for three popups telling you AW has crashed
4. Click on details to get more information

## Expected behavior
I was expecting AW to launch normally (like it did all the time prior to the BSOD).

## Documentation
Logfiles can be found in different places depending on platform:
 - Windows: `C:\Users\<USER>\AppData\Local\ActivityWatch\Logs`

[AW-Logs-19KW32.zip](https://github.com/ActivityWatch/activitywatch/files/3487297/AW-Logs-19KW32.zip)

## Environment
 - ActivityWatch version: v0.8.0b9
- Windows 10, v1809 (Build 17763.652) with latest updates

System specifications:
CPU: Ryzen 5 3600 (Stock cooler, no OC)
GPU: Saphhire RX 580 8GB (Radeon Software Version 19.6.2)
MBD: GA-AB350-Gaming 3 (B350 Chipset, BIOS version F42a, Update AGESA 1.0.0.3 ABB)
RAM: 2x8GB G.Skill RipJaws V DDR4-3200 (running at 3200 MHz)

## Additional context
Just before AW stopped working, my PC got a BSOD with THREAD_STUCK_IN_DEVICE_DRIVER after trying to access Netflix through the Edge browser (the Windows 10 app works just fine) and playing back a video. As soon as I hit play, the BSOD came up. After restarting my machine, AW stopped working. I downloaded the latest version of AW from GitHub's releases page and replaced all the files, but AW still won't boot up.

**Comment by KeksXtreme** [comment]
Even though I restarted my machine (using Windows -> restart) a couple of times before posting this bug report, now my problems seem to have vanished and ActivityWatch works just fine again.

**Comment by johan-bjareholt** [comment]
> Even though I restarted my machine (using Windows -> restart) a couple of times before posting this bug report, now my problems seem to have vanished and ActivityWatch works just fine again.

I don't get it, why did you remove this comment and why did you reopen this issue?

The log you posted does not contain any error messages which is odd, without more information we can't know what has gone wrong.
If you get a BSOD there's a risk that some data on your harddrive got corrupted, if that's the case and you have no backup there's no easy to restore the aw-server data and you have to delete the aw-server database file to make it work again.

**Comment by KeksXtreme** [comment]
I was a bit hasty when writing the comment as ActivityWatch (AW) had the taskbar icon up and running. Later I switched to the localhost website and found out that it still didn't work.

"aw-server.ini" contained a bunch of zeroes, same with other files in AW's AppData folder. Renaming/deleting that whole folder in AppData made AW work again. Thanks!

**Comment by schackbrian2012** [comment]
@vschafflik Unfortunately, I am experiencing a similar issue to you. When I restart my computer, I get the error message, "Module aw-watcher-window quit unexpectedly." It has three buttons: Hide Details..., Cancel, and Restart. The details pane says the following:

```
2023-07-03 12:33:23 [INFO ]: aw-watcher-window started  (aw_watcher_window.main:62)
2023-07-03 12:33:24 [INFO ]: Using swift strategy, calling out to swift binary  (aw_watcher_window.main:67)
2023-07-03 12:33:24 [INFO ]: Connection to aw-server established by aw-watcher-window  (aw_client.client:436)
```

You have Windows, and I have macOS. Since you had success deleting the AppData folder, I tried deleting the Application Support folder at ~/Library/Application\ Support/activitywatch. Unfortunately, when I restarted the computer, I got the same error message. In the Chrome web browser, I navigated to ActivityWatch –> Open web UI –> Activity. Unfortunately, it says, "No data." How can I prevent this error from occurring and get data to show up?
